assets/training/distillation/components/data_generation/asset.yaml

```diff
@@ -0,0 +1,3 @@
+type: component
+spec: spec.yaml
+categories: ["Foundational Models", "Finetune"]
```

Comment:
May be add distillation also as a category

Reply:
Done